item-spec/examples/CBERS_4_MUX_20181029_177_106_L4.json

```diff
@@ -49,15 +49,15 @@
   "links": [
     {
       "rel": "self",
-      "href": "https://cbers-stac-0-6.s3.amazonaws.com/CBERS4/MUX/177/106/CBERS_4_MUX_20181029_177_106_L4.json"
```

Comment:
Maybe the example should show the self link as a absolute URL since thats whats recommended in the description below.

Reply:
I agree, I chose to use absolute path as an example of a possible way to write the item but since the same item has absolute URLs it contradicts the recommendation. It would be better to use absolute paths for all references, representing the case where the data is not online,